### PR TITLE
Revised BOM again

### PR DESCRIPTION
**Pull request by triller-telekom**
Sync with https://github.com/openhab/openhab-distro/pull/936

Signed-off-by: Stefan Triller <stefan.triller@telekom.de>

### CODE REVIEW

File: bom/pom.xml [hunk 1]
```diff
@@ -4,55 +4,55 @@
...
     <dependencies>
       <dependency>
-        <groupId>org.openhab.addons.bundles</groupId>
+        <groupId>org.openhab.addons.bom</groupId>
         <artifactId>org.openhab.binding.zigbee</artifactId>
         <version>${project.version}</version>
       </dependency>
       <dependency>
-        <groupId>org.openhab.addons.bundles</groupId>
+        <groupId>org.openhab.addons.bom</groupId>
```
Comment (wborn) [inline bom/pom.xml]: The dependency groupIds shouldn't be changed. The BOM should list the bundles that this repo provides.

```suggestion
        <groupId>org.openhab.addons.bundles</groupId>
```

File: bom/pom.xml [hunk 2]
```diff
@@ -4,55 +4,55 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
-    <groupId>org.openhab.addons.bundles</groupId>
+    <groupId>org.openhab.addons.bom</groupId>
```
Comment (wborn) [inline bom/pom.xml]: It should reference the GAV parameters of the parent POM. With these changes it refers to a parent POM that doesn't exist.

Instead you could override the groupId tag from the parent by adding a new `<groupId>org.openhab.addons.bom</groupId>` 
above the
`<artifactId>org.openhab.addons.bom.zigbee</artifactId>`
below. :arrow_down:

**Comment by triller-telekom** [comment]
Thanks for your review again, I have done the changes.

**Comment by cdjackson** [comment]
Thanks guys.

**Pull request merged**